### PR TITLE
Added `hover` effect

### PR DESCRIPTION
**Pull request by Franqsanz**
Added the `hover` effect and a scss variable.

### CODE REVIEW

File: assets/sass/base/_base.scss [hunk 1]
```diff
@@ -70,6 +70,10 @@ h5 {
     }
```
Comment (makermelissa) [inline assets/sass/base/_base.scss:70]:
```suggestion
    &:hover {
        background-color: darken($purple, 15%);
    }
}
```

**Review by makermelissa — CHANGES_REQUESTED** [review]
You should be able to just add a hover to the button class and use the sass darken function instead of creating another variable.

File: assets/sass/base/_base.scss (hunk 1 above)
Comment (Franqsanz) [reply assets/sass/base/_base.scss:70]: You're right! I fix it now.

**Review by makermelissa — APPROVED** [review]
Bracing got a bit messed up, but I fixed it. Thank you for this.

**Comment by makermelissa** [comment]
No worries. That's how we learn. It's great to have other contributing.

**Comment by Franqsanz** [comment]
Thank you very much!

**Pull request merged**